fetch.bs

```diff
@@ -6074,6 +6074,11 @@ that RFC's normative processing requirements to be compatible with deployed cont
  <li><p>Let <var>mimeType</var> be the result of <a>collecting a sequence of code points</a> that
  are not equal to U+002C (,), given <var>position</var>.
 
+ <li>
+  <p><a>Strip leading and trailing ASCII whitespace</a> from <var>mimeType</var>.
+
+  <p class="note">This will only remove U+0020 <a>code points</a>, if any.
+
  <li><p>If <var>position</var> is past the end of <var>input</var>, then return failure.
```

Comment:
There are no tests that hit this condition.

Reply:
Good catch, I added some.